autoload/ale/handlers/remark_lint.vim

```diff
@@ -0,0 +1,43 @@
+" Description: remark-lint for Markdown files
```

Comment:
Could you move these functions back to where they were before? These changes aren't needed.